app/views/filter/conditionPanel.scala.html

```diff
@@ -1,57 +1,28 @@
 @import org.ada.server.models.Filter
-@import org.incal.core.FilterCondition
-@import org.ada.web.util.shorten
+@import org.ada.web.util.{conditionTypeToReadableOperator, valueOrUndefined}
 
 @(
     filter: Option[Filter],
     filterLabel: String = "Filter"
 )
 
-@valueOrUndefined(condition: FilterCondition) = @{
-    val valueOrLabel =
-        condition.valueLabel match {
-            case Some(label) => Some(label)
-            case None => condition.value
-        }
-
-    valueOrLabel match {
-        case None => "undefined"
-        case Some(s) if s.isEmpty => "undefined"
-        case Some(s) => shorten(s, 80)
-    }
-}
-
-@defining((filter.isEmpty || filter.get._id.isEmpty) && filter.map(_.conditions.size).getOrElse(0) < 4) { showPanel =>
```

Comment:
So a full filter is always displayed regardless of the number of conditions... I think it's good but might be revisited if the filter area gets too big.

Reply:
Indeed, for long names we need to trim. I will do this as part of ada-discovery/ada-issues#144